<?xml version="1.0"?>
<launch>
  <arg name="gui" default="true"/>
  <arg name="paused" default="false"/>
  <arg name="verbose" default="false"/>
  <arg name="debug" default="false"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find ekg_auv_testing)/worlds/water_sim.world"/>
    <arg name="extra_gazebo_args" value=""/>
    <arg name="paused" value="$(arg paused)"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="$(arg gui)"/>
    <arg name="headless" value="false"/>
    <arg name="debug" value="$(arg debug)"/>
    <arg name="verbose" value="$(arg verbose)"/>
  </include>

  <!-- Beacons -->
  <include file="$(find ekg_auv_testing)/launch/uploads/beacon_config.launch">
    <arg name="robot_name" value="beacon_1"/>
    <arg name="rx_id" value="1"/>
    <arg name="transceiver_id" value="141"/>
    <arg name="beacon_type" value="X150"/>
    <arg name="x" value="0"/>
    <arg name="y" value="0"/>
    <arg name="z" value="-0.2"/>
  </include>

  <include file="$(find ekg_auv_testing)/launch/uploads/beacon_config.launch">
    <arg name="robot_name" value="beacon_2"/>
    <arg name="rx_id" value="2"/>
    <arg name="transceiver_id" value="142"/>
    <arg name="beacon_type" value="X150"/>
    <arg name="x" value="0"/>
    <arg name="y" value="20"/>
    <arg name="z" value="-0.2"/>
  </include>

</launch>
